packages/hardhat/hardhat.config.ts

```diff
@@ -76,6 +77,11 @@ const config: HardhatUserConfig = {
       url: `https://polygon-mumbai.g.alchemy.com/v2/${providerApiKey}`,
       accounts: [deployerPrivateKey],
     },
+    zkSyncTestnet: {
+      url: "https://testnet.era.zksync.dev",
+      zksync: true,
+      accounts: [deployerPrivateKey],
+    },
```

Comment:
Is there a mainnet also?

Reply:
What about the `ethNetwork` prop that I see in the docs?

Reply:
Yeah, there is a mainnet as well, I think it works the same way and we would only need to add the proper configs according to [this](https://era.zksync.io/docs/reference/troubleshooting/important-links.html).
However, I didn't want to use real ETH to test it, and didn't want to add something that might be broken.

With ```ethNetwork``` we could deposit founds from the mainnet/goerli, but I it would make to code more complex, it's easier if we already bridged founds.